jscomp/core/js_main.ml

```diff
@@ -84,9 +84,23 @@ let define_variable s =
 
   
 let buckle_script_flags =
+  ("-bs-super-errors",
+    Arg.Unit (fun _ -> 
+      (* needs to be set here instead of, say, setting a
+        Js_config.better_errors flag; otherwise, when `anonymous` runs, we
+        don't have time to set the custom printer before it starts outputting
+        warnings *)
+      Super_location.setup ();
+      Super_warnings.setup ();
+      Super_typetexp.setup ();
+      Super_typecore.setup ();
```

Comment:
Didn't want to call it better-errors to avoid controversies about the "better" part